sqlparser/ast_methods.go

```diff
@@ -871,10 +880,14 @@ func (node *Execute) walkSubtree(visit Visit) error {
 
 // Format formats the node.
 func (node *Prepare) Format(buf *TrackedBuffer) {
-	if node.ColumnTypes != nil {
-		buf.Myprintf("prepare %v (%v) from (as) %v", node.PreparedStatementName, node.ColumnTypes, node.PreparedStatementQuery)
+	switch buf.dialect.(type) {
+	case *mysql.MySQLDialect:
+		if node.ColumnTypes != nil {
+			buf.Myprintf("prepare %v (%v) from (as) %v", node.PreparedStatementName, node.ColumnTypes, node.PreparedStatementQuery)
```

Comment:
does mysql support any columns for prepare?
https://dev.mysql.com/doc/refman/8.0/en/prepare.html - looks like it expects only statement name and then query after `from`

Reply:
plus as I see, we don't render anything in case of `if node.ColumnTypes == nil` and haven't even fallthrough to execute pgsql's logic